android/src/main/java/com/amplitude/android/plugins/AndroidContextPlugin.kt

```diff
@@ -30,7 +30,7 @@ class AndroidContextPlugin : Plugin {
         return event
     }
 
-    private fun initializeDeviceId(configuration: Configuration) {
+    fun initializeDeviceId(configuration: Configuration) {
```

Comment:
Question: Should this be protected?

Reply:
Thanks for reviewing! I think it is the same, tried adding `protected`, IDE shows `'protected' visibility is effectively 'private' in a final class`.

Reply:
internal?